src/sagemaker/session.py

```diff
@@ -93,7 +92,9 @@ def _initialize(self, boto_session, sagemaker_client, sagemaker_runtime_client):
         self.sagemaker_client = sagemaker_client or self.boto_session.client('sagemaker')
         prepend_user_agent(self.sagemaker_client)
 
-        self.sagemaker_runtime_client = sagemaker_runtime_client or self.boto_session.client('runtime.sagemaker')
+        config = botocore.config.Config(read_timeout=80)
+        self.sagemaker_runtime_client = sagemaker_runtime_client or self.boto_session.client('runtime.sagemaker',
+                                                                                             config=config)
```

Comment:
it'd be better to create `config` only when needed (i.e. only when `sagemaker_runtime_client` is `None`), and since there's an awkward line break with this change now, I think I'm more inclined toward

```python
if sagemaker_runtime_client is not None:
    self.sagemaker_runtime_client = sagemaker_runtime_client
else:
    config = botocore.config.Config(read_timeout=80)
    self.sagemaker_runtime_client = self.boto_session.client('runtime.sagemaker', config=config)
```

personal preference, though, and not a strong one at that.

Reply:
Good point. Updating.